Makefile

```
@@ -92,42 +92,38 @@ bench-rust: ## Run Rust benchmarks
 
 # Linting
 
-lint: clippy
-
-clippy: ## Run cargo-clippy to lint Rust code
+lint-rust: ## Run cargo-clippy to lint Rust code
 	cargo clippy --all --all-targets --all-features -- -D warnings
 
-ktlint: ## Run ktlint to lint Kotlin code
+lint-kt: ## Run ktlint to lint Kotlin code
```

Let's call it `kotlin`

```suggestion
lint-kotlin: ## Run ktlint to lint Kotlin code
```